thinc/tests/layers/test_layers_api.py

```diff
@@ -8,7 +8,7 @@
 from thinc.compat import has_torch
 import numpy
 import pytest
-
+import cupy
```

Comment:
You can't unconditionally import cupy anywhere in thinc even in the tests. Maybe this was a mistake, what were you trying to do?

Reply:
mistake